src/Api/SecretsManager/Models/Response/ProjectResponseModel.cs

```diff
@@ -5,10 +5,8 @@ namespace Bit.Api.SecretsManager.Models.Response;
 
 public class ProjectResponseModel : ResponseModel
 {
-    private const string _objectName = "project";
-
-    public ProjectResponseModel(Project project)
-        : base(_objectName)
+    public ProjectResponseModel(Project project, string obj = "project")
```

Comment:
Any reason `_objectName` was replaced by the `"project"` string literal, instead of doing the following?

```csharp
private const string _objectName = "project";

public ProjectResponseModel(Project project, string obj = _objectName)
    : base(obj)
{
    if (project == null)
    {
        throw new ArgumentNullException(nameof(project));
    }

    Id = project.Id.ToString();
    OrganizationId = project.OrganizationId.ToString();
    Name = project.Name;
    CreationDate = project.CreationDate;
    RevisionDate = project.RevisionDate;
}

public ProjectResponseModel() : base(_objectName)
{
}
```

Reply:
No reason; I like your way. Resolved in https://github.com/bitwarden/server/pull/2752/commits/a34375505303b738daf23c1d52726fe09b18e07f